packages/frontend/src/features/allertsPanel/components/Description.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import styled from "styled-components"
+
+ 
+const AllertDescTxt = styled.p`
+    font-size: 14px;
+    text-align: left;
+    line-height: 24px;
```

Comment:
jak dodajesz line-height bo był na projekcie to sprawdz czy on w ogóle wpływa na coś bo adobexd dodaje go z automatu do kazdego tekstu

Reply:
wpływa, tekst był wyraźnie od siebie odsunięty a bez line-height był zbyt zbity